### ISSUE TITLE
Seeming inability to fetch Hubzilla objects

### ISSUE THREAD

**Issue by aliceif** [dahlia/hollo#78]
Both in the admin federation page and in clients' search bar, retrieving posts from Hubzilla such as https://hub.utsukta.org/item/368dc0ca-da0a-4d16-99bd-cc1047d8ecd0 fails. Fedify lookup works though.

**Comment by dahlia** [comment]
This issue is a problem that occurs on two different layers. First, it is because Hubzilla uses an incorrect syntax of the `Link` header in the HTTP response of the actor object:

```http
Link: ...,<https://hub.utsukta.org/channel/sk>; rel="alternate"; type="application/ld+json; profile="https://www.w3.org/ns/activitystreams"",...
``` 

If you look closely, you will see that the `type` parameter value of the link <https://hub.utsukta.org/channel/sk> contains a quotation mark but has not been escaped. This header should be fixed as follows:


```http
Link: ...,<https://hub.utsukta.org/channel/sk>; rel="alternate"; type="application/ld+json; profile=\"https://www.w3.org/ns/activitystreams\"",...
```

Secondly, when Fedify encounters a `Link` header with incorrect syntax, it fails the lookup process entirely. This is not technically a bug, but it is not a robust behavior.

I will report the first issue to the Hubzilla project, and for the second issue, I will fix Fedify and release a new version, and then Hollo will be able to use the new version of Fedify.

**Comment by dahlia** [comment]
Filed an [issue about this](https://framagit.org/hubzilla/core/-/issues/1883) on the Hubzilla's issue tracker.

**Comment by dahlia** [comment]
Fixed in [Hollo 0.3.2](https://github.com/dahlia/hollo/pkgs/container/hollo/324829095?tag=0.3.2).
